debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -258,6 +258,15 @@ public static EventProcessingFailureHandlingMode parse(String value) {
             .withDescription("Optional list of custom converters that would be used instead of default ones. "
                     + "The converters are defined using '<converter.prefix>.type' config option and configured using options '<converter.prefix>.<option>'");
 
+    /** The comma-separated list of operations that snapshot reader will skip reading them from snapshot */
+    public static final Field SKIP_SNAPSHOT_OPERATIONS = Field.create("skip.snapshot.operations")
```

Comment:
The events in question originate not from snapshotting, but from streaming. How about `skipped.operations`? The expected values should be `i`, `u` and `d`, to be consistent with the values exposed for the `op` field in change events.

Reply:
it's perfect. I'm going to change it